pkg/pipeline/PipelineBuilder.go

```diff
@@ -1648,7 +1664,7 @@ func (impl PipelineBuilderImpl) BuildArtifactsForParentStage(cdPipelineId int, p
 
 func (impl PipelineBuilderImpl) BuildArtifactsForCdStage(pipelineId int, stageType bean2.WorkflowType, ciArtifacts []bean.CiArtifactBean, artifactMap map[int]int, parent bool) ([]bean.CiArtifactBean, map[int]int, error) {
```

Comment:
pass 30 for the calling function, both to this and also to BuildArtifactsForCIParent